README.md

```diff
@@ -24,6 +24,7 @@ $ markdownlint --help
     -s, --stdin                         read from STDIN (no files)
     -o, --output [outputFile]           write issues to file (no console)
     -c, --config [configFile]           configuration file (JSON or YAML)
+    -r, --rules  [file|directory|glob]  custom rule files
```

Comment:
Or packages? I think what you did for files will also support packages, so let’s say that here.

Reply:
Wait, maybe that is wrong. The new code searches for all .js files, then require-s them all separately. So packages are not supported. What if it was used like this instead: —rules file.js dir/package

Reply:
Basically, just require each of the things that was passed to —rules.